spec/models/user_spec.rb

```diff
@@ -1,11 +1,31 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  it 'is database authenticable' do
-    user = User.create(
-      email: 'test@example',
-      password: 'a7&43Wcxy6ij'
-    )
-    expect(user.valid_password?('a7&43Wcxy6ij')).to be_truthy
+  subject { FactoryBot.build(:user) }
+  describe 'validation' do
+    it 'is valid with valid attributes' do
+      expect(subject).to be_valid
+    end
```

Comment:
I ran RSpec locally, and out of sheer luck found that this is what we call a "flaky test"; it occasionally fails without any changes to the application. This is the error message I got:

```
     Failure/Error: expect(subject).to be_valid
       expected #<User id: nil, email: "edwin@labadie-larkin.test", created_at: nil, updated_at: nil> to be valid, but got errors: Password is too weak
     # ./spec/models/user_spec.rb:7:in `block (3 levels) in <main>'
```

The users factory is occasionally creating weak passwords. Due to security, the user model's password attribute isn't visible in the test, unfortunately. If you want to keep using the `strong_password` gem, you'll want to ensure that the user factory can consistently generate valid users.

Because this PR comes with another important change (Bootstrap), one thing you could do as a temporary solution is just use `xit` like in the comments below, and create a ticket to fix this test. At my workplace, I would've wanted to add Bootstrap in another PR first since it doesn't depend on Devise, but only because you've gotten this far on this branch, I think that's okay.